_rules/non-visual-reference-alternative-9bd38c.md

```diff
@@ -0,0 +1,433 @@
+```
+
+### Failed
+
+#### Failed Example 1
+
+This paragraph includes the [visual reference word][] "right". The user is told to find the menu on the right but the menu is not identified in any other way.
+
+```html
+<head>
+	<title>Failed example 1 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the menu on the right, to navigate</p>
+		</div>
+		<div class="col">
+			<ul>
+				<li>
+					<a href="https://www.w3.org/Consortium/contact">Contact</a>
+				</li>
+				<li>
+					<a href="https://www.w3.org/Help/">Help and FAQ</a>
+				</li>
+			</ul>
+		</div>
+	</div>
+</body>
+```
+
+#### Failed Example 2
+
+This paragraph includes the [visual reference word][] "right". The user is told to find the navigation on the right and the navigation is correctly identified by a `nav` element, but there are 2 `nav` elements on the page so the user doesn't know which one to use (the non-visual reference is ambiguous and does not identify a specific [web content][]).
+
+```html
+<head>
+	<title>Failed example 2 9bd38c</title>
+	<link rel="stylesheet" type="text/css" href="../test-assets/9bd38c/columns.css" />
+</head>
+<body>
+	<nav>
+		<ul>
+			<li>
+				<a href="https://www.w3.org/">W3C homepage</a>
+			</li>
+			<li>
+				<a href="https://www.w3.org/standards/">Standards</a>
+			</li>
+		</ul>
+	</nav>
+	<div class="col-container">
+		<div class="col">
+			<p>Find the navigation on the right, for the non-essential links</p>
+		</div>
+		<nav>
+			<div class="col">
+				<ul>
+					<li>
+						<a href="https://www.w3.org/Consortium/contact">Contact</a>
+					</li>
+					<li>
+						<a href="https://www.w3.org/Help/">Help and FAQ</a>
+					</li>
+				</ul>
+			</div>
+		</nav>
+	</div>
+</body>
+```
+
+#### Failed Example 3
+
+This paragraph includes the [visual reference word][] "triangle" (which is not included in the identified content) and no other indication is present so the rule fails. The fact that the triangle menu is on a different page of the same website does not restrict the rule.
+
+```html
+<body>
+	<p>
+		On the <a href="/test-assets/SC1.3.3-triangle-menu-without-heading.html">information page</a> you can find more
+		examples within the triangle menu
+	</p>
+</body>
+```
+
+#### Failed Example 4
+
+This paragraph includes the [visual reference word][] "star" (which is not included in the identified content) and there is no other indication. The content described is in an `iframe`.
+
+```html
+<body>
+	<p>More examples can be found when you look underneath the star</p>
+	<iframe src="/test-assets/SC1.3.3-star-without-heading.html"></iframe>
+</body>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+The content is indicated with the word "box" (a visual reference word), but this indication is hidden and not included in the accessibility tree because it has 'display:none'.
```

Comment:
The fact that there is a visual reference word in the text node is irrelevant to the applicability.

```suggestion
There is no text node that is visible or included in the accessibility tree because the `p` element has 'display:none'.
```

But, isn't "Howdy" in a text node, therefore making this a passed example?